nz.mega.MEGAsync.metainfo.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 Artem Polishchuk <ego.cordatus@gmail.com> -->
+<component type="desktop-application">
+  <id>nz.mega.MEGAsync</id>
+  <metadata_license>CC0-1.0</metadata_license>
+
+  <!-- MEGAsync is under a proprietary license, except the SDK which is BSD -->
+  <project_license>LicenseRef-proprietary</project_license>
+
+  <name>MEGAsync</name>
+  <provides>
+    <binary>megasync</binary>
+  </provides>
+
+  <content_rating type="oars-1.0">
+  </content_rating>
+
+  <summary>
+    Easy automated syncing between your computers and your MEGA Cloud Drive.
+  </summary>
+
+  <description>
+    <p>Easy automated syncing between your computers and your MEGA cloud drive.</p>
```

Comment:
This needs a better description.

Reply:
Fixed (i hope). BTW if to much paragraphs validate tool also complain so i hope this would enough.
`• style-invalid         : Too many <p> tags for a good description [16/10]
Validation of files failed`